packages/lesswrong/components/votes/OverallVoteAxis.tsx

```diff
@@ -14,7 +14,7 @@ const styles = (theme: ThemeType): JssStyles => ({
   overallSection: {
     display: 'inline-block',
     height: 24,
-    paddingTop: 2
+    paddingTop: isFriendlyUI ? 2.5 : 0
```

Comment:
Should this have changed the `isFriendlyUI` value?

Reply:
Yes. Other styling changes require it.